src/Components/Search Page/SearchPage.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import './SearchPage.css';
+import Card from '../Card/Card';
+
+const SearchPage = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [drinks, setDrinks] = useState([]);
+
+  const cocktailCards = drinks.map((drink) => {
+    return <Card title={drink.strDrink} img={drink.strDrinkThumb} />;
+  });
+
+  const getCocktail = (drink) => {
+    fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${drink}`)
+      .then((response) => response.json())
+      .then((drinks) => {
+        drinks.drinks ? setDrinks(drinks.drinks) : setDrinks([]);
+      });
+  };
+
+  const handleKeyPress = (event) => {
```

Comment:
I think that it's pretty universal that people submit a search by pressing enter so not really a big deal.  I'm was just wondering, does the search still fire from a mouse click?